layer/runtime/runtime_layer.cc

```diff
@@ -23,13 +23,20 @@
 #include "layer/support/layer_utils.h"
 #include "runtime_layer_data.h"
 
+#if defined(__ANDROID__)
+// https://github.com/KhronosGroup/Vulkan-Loader/blob/main/docs/LoaderLayerInterface.md#layer-interface-version-0
+# define EXPOSE_LAYER_INTERFACE_VERSION_0
+#endif
+
 namespace {
 // ----------------------------------------------------------------------------
 // Layer book-keeping information
 // ----------------------------------------------------------------------------
 
+#define LAYER_NAME "VK_LAYER_STADIA_pipeline_runtime"
+
 constexpr uint32_t kRuntimeLayerVersion = 1;
-constexpr char kLayerName[] = "VK_LAYER_STADIA_pipeline_runtime";
+constexpr char kLayerName[] = LAYER_NAME;
```

Comment:
Oops, sorry, I added the LAYER_NAME define and forgot to actually use it.

LAYER_NAME is now also used to build the entry point function names for Layer Interface Version 0 (see LAYER_NAME_FUNCTION macro).
I realized there were several issues around those macros, so I fixed them.
It requires a lot of macro gymnastics, but the intent is to have the layer name defined in one single place, because 
if we change the layer name later and forget to update the entry point names, there won't be any compilation error, but the layer won't work on Android anymore.

I hope that makes sense.